psibomana/author-quiz
# Project starter kit

**What does this PR do?**
Creates the project started kit

**How should this be manually tested?**

- Clone and set up the project using the instruction in the README.
- Use $ git checkout project-starter-kit to check out the branch containing the changes of this pull request
- Use $ yarn start to start the application
- Visit http://localhost:3000/ to see the default react application page.
